draft-ietf-rats-architecture.md

```diff
@@ -697,6 +697,25 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the Evidence is conveyed. As an example, the Attester and Verifier can
+communicate over a link that is established using a Root of Trust.  An example of such communications
+is over a cellular link, where the link is established by the Attester leveraging an enclave (e.g., SIM card).  The Verifier may have established the communications link and verified
+the presence of the Root-of-Trust in doing so, or may have received evidence that the communications
+link is anchored to a Root of Trust from an entity that established the
+communications link
+or may have received evidence that the communications link is anchored to a
+Root of Trust from an entity that established the communications link.
+In the latter case, anchoring the communications link to a Root of Trust would mean
+that this entity only established a communications link with the attesting
+device after verifying that the device had a Root of Trust.
+
+As a result, the Verifier may be able to trust the attestation evidence from the device without
+an additional endorsement or even a cryptographically-verifiable signature of the
+evidence.  Note that the appraisal policy employed by the Verifier may take into account the
+transient nature of the communications link security.  For instance, once the link is torn down
+then the Evidence may not be trusted until it is conveyed again over a re-established link.
```

Comment:
```suggestion
then the Evidence might not be trusted until it is conveyed again over a re-established link.
```

Reply:
This whole section seems to be more that of a use case that describes an architectural principle or concept than directly stating the principle or concept.  

I think the principle is:
- A conveyance protocol that is authenticated and integrity protected can be used to convey unprotected Evidence IF the following properties hold:
a) The key used to authenticate and integrity protect the conveyance channel is trusted by the Verifier to speak for the Attesting Environment that collected claims about the Target Environment.
b) The Evidence in the conveyance channel is supplied by the Attesting Environment.
c) The Root of Trust protects the key and the Attesting Environment with equivalent strength.

Reply:
Agree and I like Ned's articulation above. Maybe the PR should just say what's in Ned's principle?

Reply:
I have no objection.  It is concise and captures my original inten.  @nedmsmith - will you be taking the lead on the PR revision.